packages/sdk-components-react/src/components.ts

```diff
@@ -30,3 +30,4 @@ export { Vimeo } from "./vimeo";
 export { VimeoPreviewImage } from "./vimeo-preview-image";
 export { VimeoPlayButton } from "./vimeo-play-button";
 export { VimeoSpinner } from "./vimeo-spinner";
+export { XmlElement } from "./xml-element";
```

Comment:
I am a bit torn about this name XmlElement, it becomes a bit confusing because its a component that's called element, can't we call it just XML?

Reply:
Something that draw xml element, not an xml imo

Reply:
Let's decide later will merge as is